packages/mcp-server/src/server.ts

```diff
@@ -160,6 +160,9 @@ export async function configureServer({
             attributes: {
               "mcp.resource.name": resource.name,
               "mcp.resource.uri": url.toString(),
+              ...(context.userAgent && {
```

Comment:
we'll remove this once sentry's core instrumentation does this at the mcp.server level